### PR TITLE
Feature/style loader opt

### PR DESCRIPTION
**Pull request by johnBartos**
- Refactor src to use new simple-style-loader
- Unit test loader

see https://github.com/jwplayer/simple-style-loader/pull/1
pull https://github.com/jwplayer/jwplayer-commercial/tree/feature/style-loader-opt for commercial changes

**Before acceptance, package.json must use npm version of simple-style-loader** (it points locally now)

JW7-2472

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -187,7 +187,8 @@ module.exports = function(grunt) {
                 resolve: {
                     modulesDirectories: [
                         'src/js/',
-                        'src'
+                        'src',
+                        'node_modules'
```
Comment (robwalch) [inline Gruntfile.js]: Let's update the simple-style-loader package, then require the new version in this project. If you'd like to comment this out and add a note on working with a local package go for it.

**Comment by johnBartos** [comment]
please test this

**Comment by johnBartos** [comment]
😑  it passes in commercial, but commercial fails when built here

**Comment by johnBartos** [comment]
test this please 😑

**Comment by johnBartos** [comment]
👌

File: src/js/utils/css.js [hunk 2]
```diff
@@ -133,8 +110,9 @@ define([
     return {
         css : _css,
         style : _style,
-        clearCss : _clearCss,
+        clearCss : styleLoader.clear,
         transform : transform,
-        hexToRgba : hexToRgba
+        hexToRgba : hexToRgba,
+        styleLoader: styleLoader
```
Comment (robwalch) [inline src/js/utils/css.js]: Why return `styleLoader`? It's not used is it?

File: src/js/utils/css.js [hunk 3]
```diff
@@ -1,16 +1,9 @@
...
-    var _styleRules = {},
-        _styleSheet;
-
-    var _css = function(selector, styles) {
-        if (!_styleSheet) {
-            _styleSheet = document.createElement('style');
-            _styleSheet.type = 'text/css';
-            document.getElementsByTagName('head')[0].appendChild(_styleSheet);
-        }
+    var _css = function(playerId, selector, styles) {
```
Comment (robwalch) [inline src/js/utils/css.js]: Changing the signature of `utils.css` is a no-no, unless you are adding a new param. There are developers that use out utils even though they are undoc'ed. Make `playerId` a new param, and let the style loader append the global sheet if it's undefined.

**Comment by johnBartos** [comment]
test this please

**Comment by johnBartos** [comment]
test will fail until https://github.com/jwplayer/simple-style-loader/pull/2 is merged & npm updates

**Comment by jw-kaurand** [comment]
test this please

**Comment by johnBartos** [comment]
test this please

**Comment by johnBartos** [comment]
test this please

**Pull request merged**